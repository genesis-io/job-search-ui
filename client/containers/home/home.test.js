import React from 'react';
import Home from './home';

test('render home component', () => {
  const wrapper = shallow(<Home />);
  expect(wrapper).toMatchSnapshot();
});