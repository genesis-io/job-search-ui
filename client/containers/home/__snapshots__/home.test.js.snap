// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home component 1`] = `
<div
  className="home"
>
  <Connect(NavBar) />
  <div
    className="home-cta"
  >
    <h1
      className="home-cta-header"
    >
      Keep Track of everything for your job search
    </h1>
    <h3
      className="home-cta-sub"
    >
      Plan, Map, and execute your way to success.
    </h3>
    <div
      className="home-cta-overlay"
    />
  </div>
  <div
    className="home-info"
  >
    <InfoCard
      color="#0090d9"
      header="what in the fuck are you doing ?"
      logo="fa-chrome"
      message="stay strong"
    />
    <InfoCard
      color="#de7c01"
      header="what in the fuck are you doing ?"
      logo="fa-chrome"
      message="stay strong"
    />
  </div>
</div>
`;
